srcpkgs/numen/files/numen/conf

```diff
@@ -0,0 +1,7 @@
+# Example to run the service as a specific user:
```

Comment:
this file has to be marked as a conf_file, otherwise it would be overwritten on the next update

Reply:
Thank you :)!